examples/user_signature_from_account/main.go

```diff
@@ -0,0 +1,155 @@
+/*
+ * Flow Go SDK
+ *
+ * Copyright 2019-2020 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"github.com/onflow/cadence"
+	"google.golang.org/grpc"
+
+	"github.com/onflow/flow-go-sdk"
+	"github.com/onflow/flow-go-sdk/client"
+	"github.com/onflow/flow-go-sdk/crypto"
+	"github.com/onflow/flow-go-sdk/examples"
+)
+
+func main() {
+	UserSignatureFromAccountDemo()
+}
+
+var script = []byte(`
+import Crypto
+
+pub fun main(
+  address: Address,
+  signatures: [String],
+  keyIndexes: [Int]
+  message: String,
+): Bool {
+	let keyList = Crypto.KeyList()
+	
+	let account = getAccount(address)
+	let keys = account.keys
+	
+	var i = 0
+	while true {
+		if let key = keys.get(keyIndex: i) {
+			if key.isRevoked {
+				continue
+			}
+			keyList.add(
+				PublicKey(
+					publicKey: key.publicKey.publicKey,
+					signatureAlgorithm: SignatureAlgorithm.ECDSA_P256
```

Comment:
Better to not hardcode
```suggestion
					signatureAlgorithm: key.publicKey.signatureAlgorithm
```